all_tools_in_file.py

```diff
@@ -0,0 +1,22 @@
+from fastq_filtration import filter_fastq
+from protein_tools import run_protein_tools
+from dna_rna_tools import run_dna_rna_tools
```

Comment:
Так импорты работать не будут:)
```suggestion
from tools.fastq_filtration import filter_fastq
from tools.protein_tools import run_protein_tools
from tools.dna_rna_tools import run_dna_rna_tools
```